logging/grafana-loki/docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: "3.7"
+
+networks:
+  traefik-proxy:
+    external: true
+  loki:
+
+services:
+  loki:
+    image: grafana/loki:2.9.0
+    ports:
+      - "3100:3100"
+    command: -config.file=/etc/loki/loki-config.yaml
+    labels:
+      - traefik.enable=true
+      - traefik.http.services.loki.loadbalancer.server.port=3100
+      - traefik.http.routers.loki.rule=Host(`logging.subspace.network`)
+      - traefik.http.routers.loki.tls.certresolver=le
+      - traefik.http.routers.loki.entrypoints=websecure
+      - traefik.docker.network=traefik-proxy
+    volumes:
+      - /root/loki/conf/loki/etc/loki-config.yaml:/etc/loki/loki-config.yaml:ro
+    networks:
+      - loki
+      - traefik-proxy
+
+  promtail:
+    image: grafana/promtail:2.9.0
+    command: -config.file=/etc/promtail/config.yml
+    ports:
+      - 127.0.0.1:3200:3100
+    networks:
+      - loki
+
+  mimir:
+    image: grafana/mimir:2.9.0
+    volumes:
+      - /root/mimir/conf/etc/mimir.yml:/etc/mimir-config/mimir.yml:ro
+    entrypoint:
+      - /bin/mimir
+      - -config.file=/etc/mimir-config/mimir.yml
+    ports:
+      - "9009:9009"
+    networks:
+      - loki
+
+  agent:
+    profiles: [agent]
```

Comment:
is this pre-defined, seems weired seeing the `[]`

Reply:
This will only run agent if profile is enabled. I need this for testing now, while deciding whether to use static mode or flow mode.